packages/adapter-netlify/README.md

```diff
@@ -36,6 +36,27 @@ Then, make sure you have a [netlify.toml](https://docs.netlify.com/configure-bui
 
 It's recommended that you add the `build` and `functions` folders (or whichever other folders you specify) to your `.gitignore`.
 
-## Extra Resources
-
-Guide for SvelteKit + Netlify Forms users: https://dev.to/swyx/how-to-use-sveltekit-with-netlify-forms-5gmj
+## Troubleshooting
+
+### How to make Netlify forms work?
+
+1. Add the following [prerendering configuration](https://kit.svelte.dev/docs#configuration-prerender) to `svelte.config.js`:
+   ```js
+   prerender: {
+       crawl: true,
```

Comment:
This is a bit confusing because this is mostly specifying the defaults. I think `force: true` is the only one here that has an effect. And is that one actually required to be set? I don't understand why a Netlify form could be used only if prerendering. Is there also an alternative of just setting `prerender = true` on the page you have a form on or can we just remove this section altogether?

Reply:
I'm pretty sure that it was needed when I tested it initially. But doesn't seem to be the case anymore. Good catch!